docs/agent-browser-port-spec.md

```
@@ -265,6 +265,15 @@ cmux capabilities
 cmux browser identify --surface <surface-id>   # wrapper over system.identify + browser fields
 ```
 
+Agent MCP transport:
+```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)